reframe/frontend/cli.py

```diff
@@ -617,7 +617,8 @@ def main():
                     printer.info(runner.stats.performance_report())
 
         else:
-            printer.info('No action specified. Exiting...')
+            printer.info("No action specified. Please specify `-l'/`-L' for "
```

Comment:
I would also change both those to `printer.error()`, so as to get printed in red.

Reply:
I have also merged the message in a single line.